src/SignalR.Orleans/Extensions.cs

```diff
@@ -41,15 +39,13 @@ public static ClientConfiguration AddSignalR(this ClientConfiguration config)
         public static ISiloHostBuilder UseSignalR(this ISiloHostBuilder builder)
         {
             return builder
-                .AddApplicationPartsFromReferences(typeof(ClientGrain).Assembly)
-                .AddApplicationPartsFromReferences(typeof(GroupGrain).Assembly);
```

Comment:
Good! That was old code when I had those grains on different assemblies. Thanks!